### PR TITLE
German localization

### PR DESCRIPTION
**Pull request by technocidal**
I'm pretty happy with this one as a starting point. I'll post an additional comment on the existing German PR, so that @fweber-de can take a look and we'll probably find a way to combine our efforts.

### CODE REVIEW

File: CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff [hunk 1]
```diff
@@ -37,1032 +37,1033 @@
...
         <note/>
       </trans-unit>
       <trans-unit id="Which quick access link is shown." xml:space="preserve">
-        <source>Which quick access link is shown.</source>
+        <source>Which quick access link is shown.</source><target>Folgende Schnellverknüpfung wird angezeigt.</target>
         <note>Shown as a subtitle for a row on Settings</note>
       </trans-unit>
       <trans-unit id="Why not shows?" xml:space="preserve">
-        <source>Why not shows?</source>
+        <source>Why not shows?</source><target>???</target>
```
Comment (technocidal) [inline CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff:986]: @cliss Could you clarify the meaning/context of the source string?

Comment (cliss) [reply CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff:986]: It's here:

image

Basically, when you do a cast union search, that only works for movies, and not shows. This button is to get ahead of the inevitable support emails saying HEY HOW COME THIS DOESNT WORK FOR TV‽ 

🙃

**Comment by DonSqueak** [comment]
Hi there, I would like to fix some typos in the latest pull request; how do I do this? I have very little experience with Github...

**Comment by cliss** [comment]
Yes, you could fork this fork, and then issue a pull request to https://github.com/technocidal/callsheet-localizations/

That being said, if it's just a couple, perhaps you could drop them as comments here?

**Comment by technocidal** [comment]
@DonSqueak Thanks for your support. Feel free to issue a PR. I'll get to it today. And if there are any conflicts with the base branch, you can probably leave them up to me to fix, because I just accidentally set the wrong translation state for all keys 🤦

**Comment by DonSqueak** [comment]
So, I created the PR. Two main things: unfortunately, German is pretty strict about hyphens in compound words, so I had to go crazy on "TV-Sendungen" et al. in place of "TV Sendungen". Another: there is a mostly unknown rule that there must be a space before '...', so I fixed all these as well.

I changed "pedantisch" to "pingelig" for persnickety, as I feel this matches the intended wink-emoji better.

Rewrote a couple more and added some commas.

**Comment by DonSqueak** [comment]
Thanks for the quick merge! I also read your new amendments and I like all of them btw :)

**Comment by technocidal** [comment]
Thanks! I'll do another pass this evening and then we should be good.

**Comment by technocidal** [comment]
@cliss This PR is ready to be merged. All suggestions have been incorporated and I've completed an additional pass to check for clarity or any other stray errors.

**Comment by cliss** [comment]
This looks great; **thank you**!! If you're interested, please [reach out on Mastodon](https://mastodon.social/@caseyliss) (or [via my website](https://www.caseyliss.com/contact)) and send me a mailing address I can send an ATP sticker to. :)

**Comment by cliss** [comment]
Oh, and if y'all are interested in jumping on the TestFlight, shoot me a TestFlight-friendly email address as well!

**Comment by technocidal** [comment]
@cliss I just realized that you probably can not see my private mentions on Mastodon regarding the TestFlight invite, because Ivory does not have a separate feed for them. I'll shoot you an E-Mail.

**Pull request merged**